doc/man/8/ceph-disk.rst

```diff
@@ -15,31 +15,30 @@ Synopsis
                 ...
 
 | optional arguments:
-  -h, --help            show this help message and exit
-  -v, --verbose         be more verbose
-  --log-stdout          log to stdout
-  --prepend-to-path PATH
+|   -h, --help          show this help message and exit
+|   -v, --verbose       be more verbose
+|   --log-stdout        log to stdout
+|   --prepend-to-path PATH
                         prepend PATH to $PATH for backward compatibility (default /usr/bin)
-  --statedir PATH       directory in which ceph state is preserved (default /var/lib/ceph)
-  --sysconfdir PATH     directory in which ceph configuration files are found (default /etc/ceph)
-  --setuser USER        use the given user for subprocesses, rather than ceph or root
-  --setgroup GROUP      use the given group for subprocesses, rather than ceph or root
+|   --statedir PATH     directory in which ceph state is preserved (default /var/lib/ceph)
+|   --sysconfdir PATH   directory in which ceph configuration files are found (default /etc/ceph)
+|   --setuser USER      use the given user for subprocesses, rather than ceph or root
+|   --setgroup GROUP    use the given group for subprocesses, rather than ceph or root
```

Comment:
So, I should revoke this commit?

I still didn't get why people applied d355a383d17d9ad1dd249a2760da585d4965649d, just because it's out of maintaince? To me current version is too short, not that easy to understand.